salt/client/ssh/__init__.py

```diff
@@ -903,6 +905,9 @@ def __init__(
             self.wipe = False
         else:
             self.wipe = bool(self.opts.get("ssh_wipe"))
+        self.wipe_state = ""
```

Comment:
`wipe_state` is not a name that makes it obvious how this is used. I would expect something states such as `started`, `running`, 'finished' or sth like that, but that does not seem to be what this variable is actually for. Can we come up wtih a better name?

Reply:
`wipe_state` stands for wipe the state file after processing it. As the file name is not unique we need to clean up the files after processing them. Before the file name was static and overwritten each time it processes the state.
Is `wipe_state_file` more clear for such a case?

Reply:
I see. Yes, I think `wipe_state_file` is clearer for the name of the variable that holds the name of the state file that's getting wiped.

Reply:
`wipe_state` was renamed to `wipe_state_file`  in https://github.com/openSUSE/salt/pull/414/commits/89bab932fb7307a501b5ba0bee6f959a8c1973ae